bundles/core/src/test/resources/test-queries/graphql-requests.log

```diff
@@ -1,36 +0,0 @@
-{__type(name:"ProductAttributeFilterInput"){name,description,inputFields{name,type{name}}}}
```

Comment:
The content of this file should not be removed ... unless all our components don't do GraphQL requests any more ;-)